src/IgnoreListView.cpp

```diff
@@ -137,6 +140,21 @@ IgnoreListView::SelectionChanged()
 }
 
 
+void
+IgnoreListView::MakeEmpty()
+{
+	// This breaks the contract of BListView, but we are the only ones
+	// keeping track of the items.
+
+	int32 count = CountItems();
+	BObjectList<IgnoreListItem> list(count, true);
+	for (int32 i = 0; i < count; i++)
+		list.AddItem(dynamic_cast<IgnoreListItem*>(ItemAt(i)));
```

Comment:
Can you explain this a bit?
A new BObjectList "list" is created and filled with the items of the ignore list. But then nothing is done with it?

Reply:
Thanks for explaining. Hope I can remember that.. :)